03-edit.md

```diff
@@ -139,6 +139,8 @@ that are listed here:
     - Notebook repository (if necessary)
     - Data repository (if necessary)
     - Volume, issue and year
+    - Article number (**only** use the number provided by one of the
+      Editors-in-Chief in the submission issue).
```

Comment:
Do we need editorin-chief for that ? Maybe we can have a running issue where editors tell they'll use this number for this article, no ?